graphql/resolvers.go

```diff
@@ -1133,8 +1133,15 @@ func (r *mutationResolver) UpdateUserSettings(ctx context.Context, userSettings
 	return true, nil
 }
 
-func (r *queryResolver) User(ctx context.Context) (*restModel.APIUser, error) {
+func (r *queryResolver) User(ctx context.Context, userIdParam *string) (*restModel.APIUser, error) {
 	usr := route.MustHaveUser(ctx)
+	var err error
+	if userIdParam != nil {
+		usr, err = model.FindUserByID(*userIdParam)
+		if err != nil {
+			return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error getting user from user ID: %s", err.Error()))
```

Comment:
I think the more accurate error here will be not found, since most of the time the reason for getting into this branch will be passing a user that does not exist